Resources/views/CRUD/base_list.html.twig

```diff
@@ -35,7 +35,8 @@ file that was distributed with this source code.
 {% block list_table %}
     <div class="col-xs-12 col-md-12">
         {% set batchactions = admin.batchactions %}
-        {% if admin.hasRoute('batch') and batchactions|length %}
+        {% set show_batchactions = batchactions|length and admin.hasRoute('batch') and not app.request.isXmlHttpRequest %}
```

Comment:
I think `batchactions is not empty` should be prefered over `batchactions|length`:

```diff
- {% set show_batchactions = batchactions|length and admin.hasRoute('batch') and not app.request.isXmlHttpRequest %}
+ {% set show_batchactions = batchactions is not empty and admin.hasRoute('batch') and not app.request.isXmlHttpRequest %}
```